infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -275,6 +284,7 @@
      * Initiates the failure detection and healing detection tasks.
      */
     private void initializationTask() {
+        log.info("Initialization task ...");
```

Comment:
I would add a little bit more context to this info as in initialization of what task?

Reply:
fixed